cschwitalla/WES_Analysis/WES_Analysis.R

```diff
@@ -484,22 +524,158 @@ compute_vaf <- function(maf_obj){
 
 #necrotic
 vaf_nec = compute_vaf(necrotic_mafset)
-oncoplot(maf = necrotic_mafset, leftBarData = vaf_nec)
-
+oncoplot(maf = necrotic_mafset, leftBarData = vaf_nec, genes = c(nec_most_mut$Hugo_Symbol))
 
 #t1
 vaf_t1 = compute_vaf(t1_mafset)
-oncoplot(maf = t1_mafset, leftBarData = vaf_t1)
+oncoplot(maf = t1_mafset, leftBarData = vaf_t1, genes = c(t1_most_mut$Hugo_Symbol))
+
 
 #peritumoral
 vaf_inf = compute_vaf(peritumoral_mafset)
-oncoplot(maf = peritumoral_mafset, leftBarData = vaf_inf)
+oncoplot(maf = peritumoral_mafset, leftBarData = vaf_inf, genes = c(inf_most_mut$Hugo_Symbol))
+
 
 #ben 
-oncoplot(maf = benPBMC_mafset, draw_titv = TRUE)
+vaf_ben = compute_vaf(benPBMC_mafset)
+oncoplot(maf = benPBMC_mafset, leftBarData = vaf_ben, genes = c(ben_most_mut$Hugo_Symbol))
+
+
+#diff in mutated genes------------------------
+nec_most_mut = getGeneSummary(necrotic_mafset)[,c("Hugo_Symbol","MutatedSamples")] 
+#t1_most_mut = getGeneSummary(t1_mafset)[,c("Hugo_Symbol","MutatedSamples")] 
+#inf_most_mut = getGeneSummary(peritumoral_mafset)[,c("Hugo_Symbol","MutatedSamples")] 
+#ben_most_mut = getGeneSummary(benPBMC_mafset)[,c("Hugo_Symbol","MutatedSamples")] 
+
+
+nec_most_mut = nec_most_mut
+
+nec_mut_all = getGeneSummary(necrotic_mafset)[,c("Hugo_Symbol","MutatedSamples")] 
+t1_mut_all = getGeneSummary(t1_mafset)[,c("Hugo_Symbol","MutatedSamples")]
+inf_mut_all = getGeneSummary(peritumoral_mafset)[,c("Hugo_Symbol","MutatedSamples")] 
+ben_mut_all = getGeneSummary(benPBMC_mafset)[,c("Hugo_Symbol","MutatedSamples")] 
+
+
+nec_mut_all = getGeneSummary(necrotic_mafset)
 
 
+table(nec_mut_all$MutatedSamples)
+table(t1_mut_all$MutatedSamples)
+table(inf_mut_all$MutatedSamples)
+table(ben_mut_all$MutatedSamples)
 
+#color oncoplot 
+vc_cols = RColorBrewer::brewer.pal(n = 6, name = "Set1")
+#vc_cols = c("#3288bd", "#99d594", "#e6f598", "#fee08b", "#fc8d59", "#d53e4f")
+
+#vc_cols= c("#FF7F00" ,"#1F78B4" , "#FB9A99", "#252525","#33A02C", "#6A3D9A" )
+#vc_cols = c("#4575b4", "#91bfdb", "#e0f3f8", "#f33090", "#fc8d59", "#d73027")
+#vc_cols=c("#E41A1C", "#377EB8" ,"#4DAF4A", "#252525", "#FF7F00" ,"#FFFF33")
+
+vc_cols = c("#A00202", "#438EA9", "#436211", "#323949", "#EE6D17", "#FFC61A")
+names(vc_cols) = c(  'Frame_Shift_Del','Missense_Mutation',
+                     'Nonsense_Mutation',
+                     'Multi_Hit',
+                     'Frame_Shift_Ins',
+                     'In_Frame_Del'
+)
+
+#Tumor region color 
+regioncolor = c("#9e0142", "#fdae61", "#74add1", "#4B6C22")   #"#4B6C22") 
+  #c("#DF7D40", "#FAC941", "#49A7A9", "#8CAC57")
+names(regioncolor) = c("NEC", "T1", "INF", "BEN")
+regioncolor = setNames(c("#9e0142", "#fdae61", "#74add1", "#4B6C22") ,c("NEC", "T1", "INF", "BEN") )
+regioncolor
+#patienid color
+
+patientcolor = c("#a7c0b2",
```

Comment:
Maybe include a range with brewer.pal or include the color codings in an additional file